src/GraphQL/Extensions/GraphQLBuilderExtensions.cs

```diff
@@ -1202,23 +1202,45 @@ public static IGraphQLBuilder AddExecutionStrategy<TExecutionStrategy>(this IGra
     /// When applicable, place after calls to UseAutomaticPersistedQueries to ensure that the query document is recorded properly.
     /// </remarks>
     public static IGraphQLBuilder UseTelemetry(this IGraphQLBuilder builder, Action<GraphQLTelemetryOptions>? configure = null)
-        => UseTelemetry(builder, configure != null ? (opts, _) => configure(opts) : null);
+        => UseTelemetry<GraphQLTelemetryProvider>(builder, configure);
```

Comment:
Please update the link to semantic conventions. It moved here
https://opentelemetry.io/docs/specs/semconv/database/graphql/

Reply:
Done